docroot/modules/humsci/hs_dashboard/src/Plugin/Block/HsImportersInfoBlock.php

```diff
@@ -56,6 +56,14 @@ public static function create(ContainerInterface $container, array $configuratio
     );
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheMaxAge() {
+    // 5 minutes in seconds
+    return 300;
+  }
+
```

Comment:
I'm trying to anticipate Joe's questions...

Are these getCacheMaxAge() doing anything?  If yes, then why the extra caching elsewhere,  If no, then should we remove this chunk?

Reply:
Good question. I debated their existence. In their current context, this does nothing and can be removed. Once upon a time, I used to set a sensible default cache on all custom blocks so it would exist in the future if placed somewhere else. But in this case, it can go away as I don't see the context changing. I will remove it.